flake.nix

```diff
@@ -169,7 +162,7 @@
               cabal = "latest";
               haskell-language-server = {
                 src =
-                  if config.compiler-nix-name == "ghc8107" then
+                  if config.compiler-nix-name == "ghc96" then
```

Comment:
This conditional isn't necessary anymore, we can now track HLS latest release

Reply:
what would I make `hls-1.10` if we can track latest `hls-latest`?